python/py-crossbar/Portfile

```diff
@@ -0,0 +1,188 @@
+                    port:py${python.version}-openssl \
+                    port:py${python.version}-pyqrcode \
+                    port:py${python.version}-pytrie \
+                    port:py${python.version}-yaml \
+                    port:py${python.version}-sdnotify \
+                    port:py${python.version}-service_identity \
+                    port:py${python.version}-setproctitle \
+                    port:py${python.version}-setuptools \
+                    port:py${python.version}-treq \
+                    port:py${python.version}-txaio \
+                    port:py${python.version}-txtorcon \
+                    port:py${python.version}-u-msgpack-python \
+                    port:py${python.version}-urllib3 \
+                    port:py${python.version}-watchdog \
+                    port:py${python.version}-werkzeug \
+                    port:py${python.version}-zlmdb \
+                    port:py${python.version}-zopeinterface
+}
+
+subport crossbar {
+    name                crossbar
+
+    revision            0
+    categories          net
+
+    description         Run the crossbar.io WAMP application router as a \
+                        service.
+    long_description    {*}${description}
+
+    depends_build
+    depends_lib         port:py38-crossbar
+
+    set cb_config_path  ${prefix}/etc/${name}
+    set cb_data_path    ${prefix}/var/db/${name}
+    set cb_log_path     ${prefix}/var/log/${name}
+    set cb_share_path   ${prefix}/share/${name}
+
+    set cb_plist_path   ${prefix}/etc/LaunchDaemons/org.macports.${name}
+
+    set cb_config_file  ${cb_config_path}/config.json
+    set cb_config_src   ${cb_share_path}/config.json.example
+    set cb_log_file     ${cb_log_path}/crossbar.log
+
+    set cb_plist        org.macports.${name}.plist
+
+    set cb_bin          ${prefix}/bin/crossbar-3.8
+
+    set cb_user         _crossbar
+
+    add_users           ${cb_user} \
+                        group=${cb_user} \
+                        realname=Crossbar.io
+
+    distfiles
+
+    build {}
+
+    extract {
+        copy ${filespath}/${cb_plist} ${workpath}/
+    }
+
+    patch {
+        set cb_plist_template ${workpath}/${cb_plist}
+
+        reinplace "s|@NAME@|${name}|g"                  ${cb_plist_template}
+        reinplace "s|@USER@|${cb_user}|g"               ${cb_plist_template}
+        reinplace "s|@GROUP@|${cb_user}|g"              ${cb_plist_template}
+        reinplace "s|@DATADIR@|${cb_data_path}|g"       ${cb_plist_template}
+        reinplace "s|@BIN@|${cb_bin}|g"                 ${cb_plist_template}
+        reinplace "s|@CONF_FILE@|${cb_config_file}|g"   ${cb_plist_template}
+        reinplace "s|@LOGFILE@|${cb_log_file}|g"        ${cb_plist_template}
+    }
+
+    destroot.keepdirs-append \
+                        ${destroot}/${cb_config_path} \
+                        ${destroot}/${cb_data_path} \
+                        ${destroot}/${cb_log_path}
```

Comment:
There shouldn't be a slash after ${destroot} in these. (The subsequent cb_ variables already begin with a slash.)